parser-typechecker/src/Unison/Syntax/DeclPrinter.hs

```diff
@@ -199,8 +199,7 @@ fieldNames env r name dd = do
 
 prettyModifier :: DD.Modifier -> Pretty SyntaxText
 prettyModifier DD.Structural = fmt S.DataTypeModifier "structural"
-prettyModifier (DD.Unique _uid) =
-  fmt S.DataTypeModifier "unique" -- <> ("[" <> P.text uid <> "] ")
```

Comment:
Could you leave in this comment so we don't forget the syntax for uid :) 😅

Reply:
@aryairani yeah, sure! Done :)